test/errorHandler/ErrorHandlerSpec.scala

```diff
@@ -50,11 +50,11 @@ class ErrorHandlerSpec extends UnitSpec with GuiceOneAppPerSuite with Injecting{
       val standardErrorTemplate: Html = errorHandler.standardErrorTemplate(title, heading, message)
       val doc: Document = Jsoup.parse(standardErrorTemplate.toString())
 
-      val docTitle = doc.select("title").text()
+      val docTitle = doc.title()
       val docHeading = doc.select("#errorHeading").text()
       val docMessage = doc.select("#errorText").text()
 
-      docTitle shouldBe title + " GOV.UK"
+      docTitle shouldBe title + " - " + messages("title.pattern")
```

Comment:
This should be the output of the `messages("title.pattern")` call, not the call itself

Reply:
fixed